applications/ParticleMechanicsApplication/custom_conditions/particle_based_conditions/mpm_particle_point_load_condition.h

```diff
@@ -235,12 +235,12 @@ class MPMParticlePointLoadCondition
 
     void save( Serializer& rSerializer ) const override
     {
-        KRATOS_SERIALIZE_SAVE_BASE_CLASS( rSerializer, MPMParticlePointLoadCondition );
+        KRATOS_SERIALIZE_SAVE_BASE_CLASS( rSerializer, MPMParticleBaseLoadCondition );
```

Comment:
I think i was correct, wasn't it?

Reply:
this is what you did in the shell element. I think you have to put the base class here?

`void ShellThickElement3D3N::save(Serializer& rSerializer) const
{
    KRATOS_SERIALIZE_SAVE_BASE_CLASS(rSerializer, BaseShellElement);`

Reply:
ah sorry read it wrong, nevermind